yb-voyager/src/migassessment/sizing.go

```diff
@@ -1239,8 +1240,9 @@ func getSourceMetadata(sourceDB *sql.DB) ([]SourceDBMetadata, []SourceDBMetadata
 			   size_in_bytes,
 			   column_count 
 		FROM %v 
+		WHERE object_type IN %v
```

Comment:
nit: can use specifier - `%s` here, although for strings and int datatype %v does works as is.

Reply:
Had to update the logic slightly. Updated this as well.